tests/PHPUnit/Framework/Fixture.php

```diff
@@ -347,6 +350,11 @@ public function clearInMemoryCaches()
         // since Plugin\Manager uses getFromGlobalConfig which doesn't init the config object
     }
 
+    /**
+     * @param \Piwik\Tests\Framework\TestingEnvironmentVariables|null $testEnvironment Ignored.
+     * @param bool|false $testCaseClass Ignored.
+     * @param array $extraPluginsToLoad Ignoerd.
+     */
     public static function loadAllPlugins(TestingEnvironmentVariables $testEnvironment = null, $testCaseClass = false, $extraPluginsToLoad = array())
```

Comment:
Out of curiosity could you explain why we have the parameters but they are ignored? (rather than removing it) Is it because of BC issues?